### ISSUE TITLE
I have aggregated these unlisted potential tracking domains as i come across them

### ISSUE THREAD

**Issue by LordRishav** [StevenBlack/hosts#1668]
```
0.0.0.0 	googleoptimize.com
0.0.0.0 	gstatic.com       # Google tracker found on many sites
0.0.0.0         facebook.net
0.0.0.0 	spotify.com       # Found on many sites' embedded
0.0.0.0 	cloudflare.com 
0.0.0.0 	cloudflareinsights.com
0.0.0.0 	improving.duckduckgo.com     # Search collection on duckduckgo
0.0.0.0 	boards.4channel.org  # Popular Image board, can be in Social media
0.0.0.0 	activehosted.com
0.0.0.0 	addtoany.com
0.0.0.0 	andbeyond.media
0.0.0.0 	associates-amazon.com
0.0.0.0 	automatad.com
0.0.0.0 	bbvms.com
0.0.0.0 	cbsistatic.com
0.0.0.0 	chmod777.ltd
0.0.0.0 	cookielaw.org
0.0.0.0 	cookiepro.com
0.0.0.0 	crmservice.eu
0.0.0.0 	easypolls.net
0.0.0.0 	flourish.studio
0.0.0.0 	fontawesome.com      # Found on many sites, can potentially track activity
0.0.0.0 	gravatar.com      # Found on many sites, can potentially track activity
0.0.0.0 	gumlet.com
0.0.0.0 	hooptalk.net
0.0.0.0 	hubspot.net
0.0.0.0 	kinja-static.com
0.0.0.0 	kinja.com
0.0.0.0 	knightlab.com
0.0.0.0 	kostprice.com
0.0.0.0 	licdn.com
0.0.0.0 	lolsefti.com
0.0.0.0 	mailchimp.com
0.0.0.0 	mailerlite.com
0.0.0.0 	mashable.com
0.0.0.0 	mntzrlt.net
0.0.0.0 	msecdn.net
0.0.0.0 	netdna-cdn.com
0.0.0.0 	nsstatic.net
0.0.0.0 	ntv.io
0.0.0.0 	pjtra.com
0.0.0.0 	polyfill.io
0.0.0.0 	scroll.com
0.0.0.0 	sentry-cdn.com
0.0.0.0 	sharethis.com
0.0.0.0 	sp-prod.net
0.0.0.0 	statcounter.com
0.0.0.0 	staticflickr.com
0.0.0.0 	statuspage.io
0.0.0.0 	stripe.com
0.0.0.0 	subscribers.com
0.0.0.0 	thaickoo.net
0.0.0.0 	truste.com
0.0.0.0 	videoplayerhub.com
0.0.0.0 	vlitag.com
0.0.0.0 	vntsm.com
0.0.0.0 	waweb.com
0.0.0.0 	windows.net
0.0.0.0 	wunderground.com
0.0.0.0 	yandex.ru
0.0.0.0 	ziffstatic.com
0.0.0.0         ams-ix-2.microsoft.com
0.0.0.0         ae29-0.icr02.ams30.ntwk.msn.net      # Additional domains on duckduckgo.com, works fine without them
0.0.0.0         be-102-0.ibr01.ams30.ntwk.msn.net
0.0.0.0         be-7-0.ibr01.dub07.ntwk.msn.net
0.0.0.0         ae124-0.icr03.dub08.ntwk.msn.net
```

I don't remember the names of all the sites I found these potential trackers in, I just look at my uBlock Origin network requests on every new site and block domains

**Comment by llacb47** [comment]
Is this a joke?

**Comment by LordRishav** [comment]
@llacb47 Of course not, why would I? I have seen these trackers on multiple websites and saw that these were not listed in the hosts, so I thought of reporting it and getting other contributors' opinion on it.

**Comment by StevenBlack** [comment]
Thank you for this @LordRishav.  I've decided to decline this suggestion.  The list includes domains foundational to the internet, finance, identity, and others.

Closing.

**Comment by dnmTX** [comment]
Oh man,the first five are the BOMB 💣 

@LordRishav,just a super friendly advise that i'd recommend you to follow it:
**add** `github.com` to your list and include it in your hosts file. Thanks for doing it,much appreciated 👍

**Comment by StevenBlack** [comment]
Dan @dnmTX please try to be nice to our visitors 🙃.

**Comment by LordRishav** [comment]
@StevenBlack Maybe you could select and add the domains which are not problematic? I suggested the domains I have blocked on my side and are not included in the list, you're free to choose whichever domains you think should be in the list :)

**Comment by kekmacska** [comment]
I filtered out OP's request lol
Here are the at least somewhat harmful domains:
Most of them are just 3rd party cookies, trackers, spam sites, but there are 3-4 malware/phising sites too

0.0.0.0 activehosted.com
0.0.0.0 addtoany.com
0.0.0.0 andbeyond.media
0.0.0.0 automatad.com
0.0.0.0 bbvms.com
0.0.0.0 flourish.studio
0.0.0.0 hubspot.net
0.0.0.0 knightlab.com
0.0.0.0 lolsefti.com
0.0.0.0 mailchimp.com
0.0.0.0 msecdn.net
0.0.0.0 netdna-cdn.com
0.0.0.0 ntv.io
0.0.0.0 pjtra.com
0.0.0.0 sharethis.com
0.0.0.0 statcounter.com
0.0.0.0 statuspage.io
0.0.0.0 thaickoo.net
0.0.0.0 truste.com
0.0.0.0 videoplayerhub.com
0.0.0.0 vlitag.com
0.0.0.0 vntsm.com
0.0.0.0 windows.net
0.0.0.0 wunderground.com
0.0.0.0 ziffstatic.com

**Comment by funilrys** [comment]
@kekmacska , please avoid necrobumping. Submit a new issue.
